wiki/Beatmap/Beatmap_description/th.md

```diff
@@ -0,0 +1,21 @@
+# คำอธิบายบีทแมป
```

Comment:
```suggestion
# คำอธิบายบีทแมพ
```
Not this sh*t again, use บีทแมพ for consistency sake

Reply:
oof sorry xD